ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -0,0 +1,586 @@
+        let popoverContentViewController = UIViewController()
+        popoverContentViewController.view = tableView
+        popoverContentViewController.modalPresentationStyle = .popover
+        popoverContentViewController.popoverPresentationController?.sourceView = sender.view
+
+        present(popoverContentViewController, animated: true)
+    }
+
+    // MARK: - Item <-> View Binding
+
+    private func addBinding(_ binding: ItemBindingInfo) {
+        itemToBindingMap[binding.item] = binding
+        viewToBindingMap[binding.view] = binding
+    }
+
+    private func removeBinding(_ binding: ItemBindingInfo) {
+        itemToBindingMap.removeValue(forKey: binding.item)
+        viewToBindingMap.removeValue(forKey: binding.view)
+    }
+
+    private func clearAllItemViews(in location: ItemLocation) {
+        switch location {
+        case .heroSet:
+            heroItems.forEach {
+                if let binding = itemToBindingMap[$0] {
+                    removeBinding(binding)
+                }
+            }
+            heroCommandStack.arrangedSubviews.forEach { $0.removeFromSuperview() }
+        case .list:
+            expandedListSections.forEach {
+                $0.items.forEach {
+                    if let binding = itemToBindingMap[$0] {
+                        removeBinding(binding)
+                    }
+                }
+            }
+        }
+    }
+
+    private func createAndBindHeroCommandView(with item: CommandingItem) -> UIView {
+        let tabItem = TabBarItem(title: item.title, image: item.image, selectedImage: item.selectedImage, largeContentImage: item.largeImage)
+        let itemView = TabBarItemView(item: tabItem, showsTitle: true)
+        itemView.alwaysShowTitleBelowImage = true
+        itemView.numberOfTitleLines = 1
+        itemView.isSelected = item.isOn
+        itemView.accessibilityTraits.insert(.button)
+
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleHeroCommandTap(_:)))
+        itemView.addGestureRecognizer(tapGesture)
+
+        NSLayoutConstraint.activate([
+            itemView.heightAnchor.constraint(equalToConstant: Constants.heroButtonHeight)
+        ])
+        let widthConstraint = itemView.widthAnchor.constraint(equalToConstant: Constants.heroButtonWidth)
+        widthConstraint.isActive = !isInSheetMode
+
+        item.delegate = self
+        let binding = HeroItemBindingInfo(item: item, view: itemView, location: .heroSet, widthConstraint: widthConstraint)
+        addBinding(binding)
+
+        return itemView
+    }
+
+    private func setupTableViewCell(_ cell: TableViewCell, with item: CommandingItem) {
+        let iconView = UIImageView(image: item.image)
+        iconView.tintColor = Constants.tableViewIconTintColor
+        cell.setup(title: item.title, subtitle: "", footer: "", customView: iconView, customAccessoryView: nil, accessoryType: .none)
+        cell.bottomSeparatorType = .none
```

Comment:
i think we should check if the cell is the last cell of the section and show full border. I thought if we had multiple sections we do have full separators